### PR TITLE
Select H5P and Gravity Forms from a searchable dropdown instead of typing in ID

### LINKED ISSUE #219: List H5P content when selecting "H5P" content type

### PR DESCRIPTION
**Pull request by narendrasss**
Fixes #219

Changes the "h5p embed link" input into a combobox that lists all available h5p content in the server.

### CODE REVIEW

**Comment by wynnset** [comment]
@narendrasss the only thing remaining on this task is to use the database table to retrieve the actual title of the H5P content instead of using its url.

**Comment by narendrasss** [comment]
@wynnset I addressed the changes. From my research it seems the only way to interact with it is through pure SQL queries, which is what I've done. Please take a look.

File: classes/class.tapestry-h5p.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class TapestryH5P implements ITapestryH5P
+{
+  private $postId;
+
+  public function __construct($postId = null)
+  {
+    $this->postId = $postId;
+  }
+
+  public function get()
```
Comment (wynnset) [inline classes/class.tapestry-h5p.php]: All public PHP functions must have a comment block including description, parameters, and return

File: classes/class.tapestry-h5p.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  {
+    global $wpdb;
+    $content = $wpdb->get_results("select id, title from wp_h5p_contents");
```
Comment (wynnset) [inline classes/class.tapestry-h5p.php]: The table prefix `wp_` is not always the same on all sites, so please get the prefix dynamically using `$wpdb->prefix`.

**Pull request merged**